libs/@blockprotocol/graph/src/stdlib/subgraph/edge/link-entity.ts

```diff
@@ -113,90 +152,175 @@ export const getIncomingLinksForEntity = (
 
   return (
     Object.entries(entityEdges)
-      // Only look at edges that were created before or at the timestamp
+      // Only look at outgoing edges that were created before or within the search interval
       .filter(
-        ([edgeTimestamp, _outwardEdges]) => edgeTimestamp <= timestampString,
+        ([edgeTimestamp, _outwardEdges]) =>
+          compareBounds(
+            { kind: "inclusive", limit: edgeTimestamp },
+            searchInterval.end,
+            /** @todo - should this be end */
+            "start",
```

Comment:
Same as above, I think this is `"end"`

Reply:
Maybe we should add a function `boundIsStrictlyRightOfInterval` and use that instead.

Reply:
Handled by [Introduce strict comparison operators for intervals](https://github.com/blockprotocol/blockprotocol/pull/928/commits/3a11838b5d74494484778721270590f661a70c30)